Common/src/Stream/Worker/TaskRequestExtensions.cs

```diff
@@ -40,27 +41,27 @@ namespace ArmoniK.Core.Common.Stream.Worker;
 [PublicAPI]
 public static class TaskRequestExtensions
 {
-  public static IEnumerable<ProcessReply.Types.CreateLargeTaskRequest> ToRequestStream(this IEnumerable<TaskRequest> taskRequests,
-                                                                                       TaskOptions?                  taskOptions,
-                                                                                       int                           chunkMaxSize)
+  public static IEnumerable<CreateTaskRequest> ToRequestStream(this IEnumerable<TaskRequest> taskRequests,
+                                                               TaskOptions?                  taskOptions,
+                                                               int                           chunkMaxSize)
   {
     var fsm = new ProcessReplyCreateLargeTaskStateMachine(NullLogger.Instance);
     fsm.InitRequest();
     if (taskOptions is not null)
     {
-      yield return new ProcessReply.Types.CreateLargeTaskRequest
+      yield return new CreateTaskRequest
                    {
-                     InitRequest = new ProcessReply.Types.CreateLargeTaskRequest.Types.InitRequest
+                     InitRequest = new CreateTaskRequest.Types.InitRequest
                                    {
                                      TaskOptions = taskOptions,
                                    },
                    };
     }
     else
     {
-      yield return new ProcessReply.Types.CreateLargeTaskRequest
+      yield return new CreateTaskRequest
                    {
-                     InitRequest = new ProcessReply.Types.CreateLargeTaskRequest.Types.InitRequest(),
+                     InitRequest        = new CreateTaskRequest.Types.InitRequest(),
```

Comment:
why?  These spaces/tabulations are not event coherent with the line 54.

Reply:
Formatting issue^^ A way to check if the code is properly formatted has been merged on master. We will be enforcing it soon.